cfgrammar/src/lib/header.rs

```diff
@@ -18,19 +18,18 @@ pub enum HeaderErrorKind {
     ExpectedToken(char),
     DuplicateEntry,
 }
+// Indicates a value prefixed by an optional namespace.
+// `Foo::Bar` with optional `Foo` specified, becomes (Some(("Foo", ...)), ("Bar", ...))
+// Alternately just `Bar` alone without a namespace is represented by (None, ("Bar", ...)).
+pub type Namespaced = (Option<(String, Span)>, (String, Span));
```

Comment:
I think this should probably be a `struct`: at least I tend to get a bit confused with types where the LHS and RHS are more-or-less the same.